### PR TITLE
Slushbot

### PR DESCRIPTION
**Pull request by kristenmills**
Converts whois and lingo to hubot scripts

### CODE REVIEW

File: src/whois.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+        robot.send({ room: msg.envelope.user.name }, position + ': ' + member.first_name + ' ' +
+          member.last_name);
+      });
+    }
+  });
+}
+
+module.exports = function(robot) {
+  robot.respond(/sse whois (.+)/i, function(msg) {
```
Comment (bbesmanoff) [inline src/whois.js]: not sure if hubot automatically takes care of this, but 

``` js
'jesse whois someone'.match(/sse whois (.+)/i) // Array [ "sse whois someone", "someone" ]
```

It does get `someone`, but it also pulls part of Jesse's name...

Maybe `/^sse whois (.+)/i` or `/\bsse whois (.+)/i`?

File: src/api.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+var API = require('sse-api-client');
+if(!process.env.SSE_API_ROOT) {
+  console.log("Warning: SSE_API_ROOT not set")
```
Comment (bbesmanoff) [inline src/api.js]: This should be mentioned in the README in case it gets buried on stdout.

File: src/whois.js (hunk 1 above)
Comment (kristenmills) [inline src/whois.js]: https://github.com/github/hubot/blob/master/src/robot.coffee#L102

Hubot builds a new regex that responds to your robots name or alias so it shouldn't be an issue.

File: src/api.js (hunk 2 above)
Comment (kristenmills) [inline src/api.js]: :+1:

File: index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+var fs = require('fs');
+var path = require('path');
+
+module.exports = function (robot, scripts) {
```
Comment (bbesmanoff) [inline index.js]: Can you just make this a [script package](https://hubot.github.com/docs/scripting/#creating-a-script-package)?  Worst case, `api.js` just lives somewhere that isn't `scripts/`

Comment (kristenmills) [inline index.js]: :+1: didn't know you could do it that way. I was following what some older packages do.

**Pull request merged**